### PR TITLE
Added two Courses (C programming and Github contribution)

### PR DESCRIPTION
**Pull request by samarsrivastav**
## What does this PR do?
 I have added a C programming course in one video and a Course for contributing in Github. It is in Hindi and easy to understand for beginners.

### Why is this valuable (or not)?
     --->it is a course perfect for students as its free and easy to understand.
### How do we know it's really free?
 -->it is available in Youtube as well.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

no shortened links, please

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -64,6 +64,7 @@
 ### <a id="c"></a>C
 
 * [All C Concepts \| Hindi](https://www.youtube.com/playlist?list=PL7ersPsTyYt1d8g5qaxbE6sjWDzs4D_1v) - Saurabh Shukla
+* [C Language Tutorial for Beginners (with Notes & Practice Questions)](https://youtu.be/irqbmMNs2Bo) -apna college
```
Comment (davorpa) [inline courses/free-courses-hi.md:67]: As suggested...

```suggestion
* [C Language Tutorial for Beginners (with Notes & Practice Questions)](https://www.youtube.com/watch?v=irqbmMNs2Bo) - Apna College
```

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -119,6 +120,7 @@
 * [Complete Git Tutorials For Beginners In Hindi](https://youtube.com/playlist?list=PLu0W_9lII9agwhy658ZPA0MTStKUJTWPi) - CodeWithHarry
 * [Git & GitHub Tutorial For Beginners In Hindi](https://www.youtube.com/watch?v=gwWKnnCMQ5c) - CodeWithHarry
 * [GitHub with Visual Studio (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhigWA1mNWzwErSBIZvgOJbNc) - Rajesh Kumar, Geeky Shows
+* [How to Fix your First Bug in OPEN SOURCE (github)](https://youtu.be/fMdfDnGX9S8) -love Babbar
```
Comment (davorpa) [inline courses/free-courses-hi.md:123]: Too short video (<1h). Internet is bast so... removing...

```suggestion
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -91,6 +92,7 @@
 * [C++ Full Course \| C++ Tutorial \| Data Structures & Algorithms](https://www.youtube.com/playlist?list=PLfqMhTWNBTe0b2nM6JHVCnAkhQRGiZMSJ) - Apna College
 * [Data Structure Programs \| Hindi](https://www.youtube.com/playlist?list=PLsFNQxKNzefK_DAUwnQwBizOmcY7aDLoY) - Saurabh Shukla
 * [Data Structures and Algorithms Course in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ahIappRPN0MCAgtOu3lQjQi) - CodeWithHarry
+* [DBMS (Database Management system) Complete Playlist](https://youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) -Gate Smashers
```
Comment (davorpa) [inline courses/free-courses-hi.md:95]:
```suggestion
* [DBMS (Database Management system) Complete Playlist](https://youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers
```

**Review by davorpa — APPROVED** [review]

**Comment by samarsrivastav** [comment]
please change the tag to approved

**Comment by eshellman** [comment]
please resolve conflicts.

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**